google-cloud-tools-plugin/ultimate/src/com/google/cloud/tools/intellij/appengine/facet/impl/AppEngineJavaeeSupportContributor.java

```diff
@@ -44,7 +44,7 @@
   @Override
   public void setupFrameworkSupport(JavaeeFrameworkSupportContributionModel model) {
     AppEngineStandardFacet appEngineStandardFacet = model.getFacet(AppEngineStandardFacetType.ID);
-    if (appEngineStandardFacet == null) {
+    if (appEngineStandardFacet == null || appEngineStandardFacet.isJava8Runtime()) {
```

Comment:
Won't this cause all the other setup to be skipped too? We still want the run configs created for java 8 runtimes.

Reply:
Ah I see, fixed. PTAL